py3status/modules/xscreensaver.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+"""
+Indicator for Xscreensaver, can be toggled by left clicking.
+
+This is a partial rewrite of dpms.py. Most credit
+goes to the original author Andre Doser. This module shows the status
+of Xscreensaver and activates or deactivates it upon left click.
+
+This script is useful for people who let Xscreensaver manage dpms
+settings. Xscreensaver has its own dpms variables separate from xset.
+Dpms can be savely turned of in xset as long as xscreensaver is running.
+Settings can be managed using "xscreensaver-demo".
+
+Configuration parameters:
+    - format_off: string to display when Xscreensaver is disabled
+    - format_on : string to display when Xscreensaver is enabled
+
```

Comment:
Being picky can you remove the `- ` in these config parameters and follow them with a colon without a space
like
```
Configuration parameters:
    format_off: string to display when Xscreensaver is disabled
    format_on: string to display when Xscreensaver is enabled
```
It helps with creating modules/README.md